packages/amplify_core/pubspec.yaml

```diff
@@ -10,7 +10,8 @@ environment:
 dependencies:
   flutter:
     sdk: flutter
-  amplify_core_plugin_interface: ^0.0.1-dev.2
+  amplify_core_plugin_interface:
+    path: ../amplify_core_plugin_interface
```

Comment:
Does it make sense to do this for all of the dependencies in this list?

Reply:
This should not be necessary now that melos is incorporated.

Reply:
This is a temporary change for testing and will be changed for the final release.

Reply:
I think these can be removed now.  Melos solves this for local testing.

Reply:
removed